See ports/180350: if archivers/libzip is installed there's a conflict because
/usr/local/include ends up being included before the minizip directory inside
the port.
--- src/Sigil/CMakeLists.txt.orig	2015-04-12 15:08:13 UTC
+++ src/Sigil/CMakeLists.txt
@@ -606,11 +606,11 @@ include_directories( BEFORE
                      ${HUNSPELL_INCLUDE_DIRS}
                      ${PCRE_INCLUDE_DIRS}
                      ${tidyLib_SOURCE_DIR}
-                     ${MINIZIP_INCLUDE_DIRS}
                      ${ZLIB_INCLUDE_DIRS}
                      ${XERCESEXTENSIONS_INCLUDE_DIRS}
                      ${XERCES_INCLUDE_DIRS}
                      ${BOOST_INCLUDE_DIRS}
+                     ${MINIZIP_INCLUDE_DIRS}
                      ${CMAKE_CURRENT_SOURCE_DIR}
                      ${CMAKE_CURRENT_BINARY_DIR} )
 
